ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -5,7 +5,7 @@ import platform
 import sysconfig
 import typing
 
-import six
+import attr
```

Comment:
🤔 I can't quite make sense of this change

Reply:
bad merge conflict, thanks for calling out.

```suggestion
```